pytorch_lightning/metrics/classification/confusion_matrix.py

```diff
@@ -24,9 +24,11 @@
 
 class ConfusionMatrix(Metric):
     """
-    Computes the confusion matrix. Works with binary, multiclass, and multilabel data.
-    Accepts logits from a model output or integer class values in prediction.
-    Works with multi-dimensional preds and target.
+    Computes the `confusion matrix
+    <https://en.wikipedia.org/wiki/Confusion_matrix>`_.  Works with binary,
```

Comment:
We cannot use link to wikipedia for confusion matrix because they describe it as preds=rows and true=columns where we follow sklearns convention preds=columns and true=rows. Instead we should either link to sklearn https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html#sklearn.metrics.confusion_matrix or rewrite the text ourself

Reply:
Will do!